package.json

```diff
@@ -1,94 +1,183 @@
-    "author": "Einar S. Idsø",
-    "license": "MIT",
-    "repository": {
-        "type": "git",
-        "url": "https://github.com/equinor/vscode-septic"
-    },
-    "version": "0.5.0",
-    "icon": "images/SEPTIC_logo.jfif",
-    "engines": {
-        "vscode": "^1.72.0"
-    },
-    "categories": [
-        "Programming Languages",
-        "Snippets"
-    ],
-    "activationEvents": [
-        "onLanguage:septic"
-    ],
-    "main": "./out/extension.js",
-    "contributes": {
-        "languages": [
-            {
-                "id": "septic",
-                "aliases": [
-                    "Septic"
-                ],
-                "extensions": [
-                    ".cnfg"
-                ],
-                "configuration": "./language-configuration.json"
-            },
-            {
-                "id": "jinja",
-                "aliases": [
-                    "Jinja Raw",
-                    "jinja-raw"
-                ],
-                "extensions": [],
-                "configuration": "./language-configuration.json"
-            }
-        ],
-        "grammars": [
-            {
-                "language": "septic",
-                "scopeName": "source.septic",
-                "path": "./syntaxes/septic.tmLanguage.json"
-            },
-            {
-                "language": "jinja",
-                "scopeName": "source.jinja",
-                "path": "./syntaxes/jinja.tmLanguage.json"
-            }
-        ],
-        "snippets": [
-            {
-                "language": "septic",
-                "path": "snippets/snippets.json"
-            }
-        ],
-        "configurationDefaults": {
-            "[septic]": {
-                "files.encoding": "windows1252",
-                "editor.wordWrap": "on",
-                "editor.insertSpaces": true
-            }
-        }
-    },
-    "scripts": {
-        "vscode:prepublish": "npm run compile",
-        "compile": "tsc -p ./",
-        "watch": "tsc -watch -p ./",
-        "pretest": "npm run compile",
-        "lint": "eslint src --ext ts",
-        "test": "node ./out/test/runTest.js"
-    },
-    "devDependencies": {
-        "@types/glob": "^8.0.0",
-        "@types/mocha": "10.0.0",
-        "@types/node": "16.x",
-        "@types/vscode": "^1.72.0",
-        "@typescript-eslint/eslint-plugin": "^5.40.0",
-        "@typescript-eslint/parser": "^5.40.0",
-        "@vscode/test-electron": "^2.1.5",
-        "eslint": "^8.24.0",
-        "glob": "^8.0.3",
-        "mocha": "^10.0.0",
-        "typescript": "^4.8.4"
-    }
+	"name": "septic-config",
+	"displayName": "Septic Language Basics",
+	"icon": "images/SEPTIC_logo.jfif",
+	"description": "Provides snippets, syntax highlighting and multi-level folding in Septic config files.",
+	"author": "Einar S Idso",
+	"license": "MIT",
+	"version": "0.5.0",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/equinor/vscode-septic"
+	},
+	"publisher": "EinarSIdso",
```

Comment:
I'm looking into transferring from my personal account to an Equinor account. May not be entirely trivial, and may take a while to get done properly.